apps/fe/src/styles/globals.css

```
@@ -84,49 +95,53 @@
 
-body {
-  background: var(--background);
-  color: var(--foreground);
-  font-family: 'Pretendard', sans-serif;
-  font-size: 1.6rem;
-  line-height: 1.5;
-  -webkit-font-smoothing: antialiased;
+  body {
+    background: var(--background);
+    color: var(--foreground);
+    font-family: 'Pretendard', sans-serif;
```

Stylelint (17.9.0)

[error] Expected no quotes around "Pretendard" (font-family-name-quotes)

(font-family-name-quotes)